support-frontend/assets/pages/contributions-landing/contributionsLandingInit.js

```diff
@@ -185,14 +186,6 @@ const init = (store: Store<State, Action, Function>) => {
 
   initialisePaymentMethods(state, dispatch);
 
-  // This will be in window.guardian if it has come from a PayPal one-off contribution,
-  // where it is returned by the Payment API to the backend, flashed into the session to preserve
-  // it through a serverside redirect, and then written into window.guardian on the thank-you page.
-  if (window.guardian.guestAccountCreationToken) {
-    dispatch(setGuestAccountCreationToken(window.guardian.guestAccountCreationToken));
-    dispatch(setThankYouPageStage('thankYouSetPassword'));
```

Comment:
does the thank you page just have one "default" stage now?

Reply:
yes, the ThankYouPageStage is a legacy thing that could have been removed previously. We used to have 2 pages, but now it's all on a single page and doesn't require this state transition.